model/accounts_mgmt/v1/billing_models_resource.model

```diff
@@ -0,0 +1,16 @@
+resource BillingModels{
```

Comment:
@tirthct  I believe you need to update root_resource.model too

Reply:
Updated root_resource model file